tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration3_8.cs

```diff
@@ -0,0 +1,74 @@
+// <copyright file="TestMethodRunnerExecuteIntegration3_8.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+using System;
+using System.Collections;
+using System.ComponentModel;
+using Datadog.Trace.Ci;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2;
+
+/// <summary>
+/// Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution.TestMethodRunner.Execute calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter",
+    TypeName = "Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Execution.TestMethodRunner",
+    MethodName = "Execute",
+    ReturnTypeName = "Microsoft.VisualStudio.TestTools.UnitTesting.TestResult[]",
+    MinimumVersion = "14.0.0",
+    MaximumVersion = "14.*.*",
```

Comment:
You've used `15.*.*` in _some_ of the other integrations - I assume this is correct as 14?